plugins/org.python.pydev.parser/tests/org/python/pydev/parser/PyParser38Test.java

```diff
@@ -194,4 +195,18 @@ public void testIterableUnpackYield() {
         parseLegalDocStr("def parse():\n    yield ()");
     }
 
+    public void testTypeDeclaration() {
+        SimpleNode node = parseLegalDocStr("attribute: str");
+        assertEquals(
```

Comment:
Comparing with a string in this case isn't ideal (if the AST gets changed for unrelated things we'd have to update the tests).

So, please compare doing `instanceof`/`typecasts`/`check attributes` as appropriate here.